spec/apollo-server-spec.yaml

```diff
@@ -0,0 +1,56 @@
+swagger: "2.0"
+info:
+  description: "This is the backend component API. Visit the link for an interactive playground to execute requests."
+  version: "1.0.0"
+  title: "Road Rules Search Engine"
+tags:
+  - name: "Backend service"
+schemes:
+  - "https"
+paths:
+  /:
+    post:
+      tags:
+        - "Root"
+      summary: "Base path"
+      description: "Base path handles any incoming request and re-routes it to the requested resolver."
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - in: "body"
+          name: "body"
+          description: "Any GraphQL request ( query/mutation/subscription )"
+          schema:
+            $ref: '#/definitions/Request'
+      responses:
+        "200":
+          description: "Default response format"
+          schema:
+            $ref: '#/definitions/Response'
```

Comment:
Are there any other responses that could happen? Or will it always be a `200` response?

Reply:
It always defaults to 200. It's a GraphQL standard. Rather than dealing with bunch of codes (and having to handle them separately), whenever an error happens, the **json** response comes with an `errors` attribute containing all errors that occurred during the request.